service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/StandardDirectionsOrderController.java

```diff
@@ -137,31 +135,20 @@ public CallbackResponse handleMidEventPrePopulation(@RequestBody CallbackRequest
         CaseData caseData = getCaseData(caseDetails);
 
         GatekeepingOrderRoute route = caseData.getSdoRouter();
-        if (URGENT != route) {
-            if (SERVICE == route) {
-                caseDetails.getData().put(
-                    DATE_OF_ISSUE_KEY, sdoService.generateDateOfIssue(caseData.getStandardDirectionOrder())
-                );
-            }
+        if (SERVICE == route) {
```

Comment:
Do we need to make changes to this controller?  It looks like the event draftSDO is deprecated.

Reply:
where is it deprecated?